egs/librispeech/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -8,20 +8,21 @@
 stage=0
 generate_alignments=true # false if doing ctc training
 speed_perturb=true
+cleanup_affix=cleaned
 
 set -e
 . ./cmd.sh
 . ./path.sh
 . ./utils/parse_options.sh
 
 
-train_set=train_960
+train_set=train_960_${cleanup_affix}    # you might set this to e.g. train_960
 if [ "$speed_perturb" == "true" ]; then
   if [ $stage -le 1 ]; then
     #Although the nnet will be trained by high resolution data, we still have to perturbe the normal data to get the alignment
     # _sp stands for speed-perturbed
 
-    for datadir in train_960; do
+    for datadir in ${train_set}; do
       utils/perturb_data_dir_speed.sh 0.9 data/${datadir} data/temp1
```

Comment:
Yes, probably the best/cleanest example of an up-to-date chain-training
script is in tedlium s5_r2.


On Mon, Aug 22, 2016 at 4:52 PM, Vijayaditya Peddinti <
notifications@github.com> wrote:

> In egs/librispeech/s5/local/nnet3/run_ivector_common.sh
> <https://github.com/kaldi-asr/kaldi/pull/991#discussion_r75756277>:
>
> >  if [ "$speed_perturb" == "true" ]; then
> >    if [ $stage -le 1 ]; then
> >      #Although the nnet will be trained by high resolution data, we still have to perturbe the normal data to get the alignment
> >      # _sp stands for speed-perturbed
> >
> > -    for datadir in train_960; do
> > +    for datadir in ${train_set}; do
> >        utils/perturb_data_dir_speed.sh 0.9 data/${datadir} data/temp1
>
> Also remember to add volume perturbation to the script. You can use
> utils/data/perturb_data_dir_volume.sh for this. I think examples for
> using both these scripts can be found in latest Tedlium/AMI recipes.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/991/files/51373685879bf008bd749c8351b4220e24096085#r75756277>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ADJVu6fXribq_SMCqFV_haNNJyYDBPRiks5qigv4gaJpZM4Jpirx>
> .
>